--- a/src/catch2/meson.build
+++ b/src/catch2/meson.build
@@ -4,8 +4,6 @@
 #        https://www.boost.org/LICENSE_1_0.txt)
 
 # SPDX-License-Identifier: BSL-1.0
-pkg = import('pkgconfig')
-
 conf_data = configuration_data()
 conf_data.set('CATCH_CONFIG_DEFAULT_REPORTER', 'console')
 conf_data.set('CATCH_CONFIG_CONSOLE_WIDTH', '80')
@@ -14,7 +12,6 @@
   input: 'catch_user_config.hpp.in',
   output: 'catch_user_config.hpp',
   format: 'cmake@',
-  install_dir: get_option('includedir') / 'catch2',
   configuration: conf_data,
 )
 
@@ -319,29 +316,8 @@
   'reporters/catch_reporter_xml.cpp',
 )
 
-headers = benchmark_headers + internal_headers + reporter_headers
 sources = benchmark_sources + internal_sources + reporter_sources
 
-# The headers must be installed with their full paths, which meson
-# v0.63 supports using `preserve_path` for `install_headers`. We'd
-# like to be compatible with Debian 11 (current stable) which has
-# meson v0.56.2. Instead, let's use the technique from
-# https://github.com/mesonbuild/meson/issues/14 with some tweaks to
-# make it compatible with newer meson.
-include_subdir = 'catch2'
-foreach file : headers
-  file_path = file.split('/')
-
-  folder = ''
-  foreach path : file_path
-    if path != file_path[-1]
-      folder = folder / path
-    endif
-  endforeach
-
-  install_headers(file, subdir: join_paths(include_subdir, folder))
-endforeach
-
 catch2_dependencies = []
 # Check if this is an Android NDK build.
 if ((host_machine.system() == 'android') or
@@ -356,7 +332,6 @@
   sources,
   dependencies: catch2_dependencies,
   include_directories: '..',
-  install: true,
 )
 
 catch2_dep = declare_dependency(
@@ -364,29 +339,14 @@
   include_directories: '..',
 )
 
-pkg.generate(
-  catch2,
-  filebase: 'catch2',
-  description: 'A modern, C++-native, test framework for C++14 and above',
-  url: 'https://github.com/catchorg/Catch2',
-)
-
 catch2_with_main = static_library(
   'Catch2Main',
   'internal/catch_main.cpp',
   link_with: catch2,
   include_directories: '..',
-  install: true,
 )
 
 catch2_with_main_dep = declare_dependency(
   link_with: [catch2, catch2_with_main],
   include_directories: '..',
 )
-
-pkg.generate(
-  catch2_with_main,
-  filebase: 'catch2-with-main',
-  description: 'A modern, C++-native, test framework for C++14 and above (links in default main)',
-  requires: 'catch2 = ' + meson.project_version(),
-)
