--- a/meson.build
+++ b/meson.build
@@ -29,12 +29,6 @@
     override_options: 'cpp_std=c++14',
     include_directories: 'include',
     build_by_default: true,
-    install: true,
-  )
-  install_subdir(
-    'include',
-    strip_directory: true,
-    install_dir: get_option('includedir'),
   )
   fmt_dep = declare_dependency(
     include_directories: 'include',
@@ -44,10 +38,4 @@
   if meson.version().version_compare('>=0.54.0')
     meson.override_dependency('fmt', fmt_dep)
   endif
-  pkg_mod = import('pkgconfig')
-  pkg_mod.generate(
-    fmt_lib,
-    name: 'fmt',
-    filebase: 'fmt',
-  )
 endif
